### ISSUE TITLE
System restarting if lock screen with activity manager is open

### ISSUE THREAD

**Issue by greds94** [fedora-silverblue/issue-tracker#591]
As I write in the title, if I lock the screen with activity manager open Fedora restart the system.

With this situation
![Screenshot from 2024-08-28 12-38-03](https://github.com/user-attachments/assets/17d3edcf-fd7e-4342-a627-3742accb623a)

If I lock with super + L, Fedora restart the system

**Comment by travier** [comment]
Can you try to reproduce the issue in a VM or a classic Workstation system? This is likely not a Silverblue specific issue and should be reported upstream.

Is this Fedora Silverblue 40?

**Comment by greds94** [comment]
It is Fedora Silverblue 40.
The workstation works fine because, before the upgrade from 39 to 40, there was no this problem.

All the office notebooks with Fedora SB 40 have this problem.

**Comment by greds94** [comment]
UPDATE: The same issue appears also in Fedora 41 SB.

**Comment by travier** [comment]
Is this still and issue on Fedora 42?

**Comment by travier** [comment]
Yes, you can open an issue for https://gitlab.gnome.org/GNOME/mutter/-/issues upstream. Please cross link the issue here.

**Comment by travier** [comment]
Can you detail more precisely what you're doing to trigger this?
